pkg/kubelet/cm/cgroupManager_mock.go

```diff
@@ -0,0 +1,45 @@
+// +build linux
```

Comment:
@Random-Liu suggested that we might also have the types definition in kubelet types package. That would solve the problem as we wont need to import cm package in the testing/mock.go file. 
But I was not sure of having the types.go in the kubelet types as of yet. Later on if this code is being used at a lot of other places then we might want to do this. @yujuhong